test-suite/squidconf/pp-ifs.conf

```
@@ -0,0 +1,27 @@
+## Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+if true
+    if 10 = 0xA
+        if 0${process_number} = 077777777
+        else
+            if 00=000
+                if  -00= +00
```

When Tokenizer::int64() can parse "0", most leading zeros will be gone:

```suggestion
        if ${process_number} = 9223372036854775807
        else
            if 0=000
                if  -0= +0
```

---

> When Tokenizer::int64() can parse "0", most leading zeros will be gone

I merged master/v7 commit 8cc5e88c into the branch and removed those distracting zeros as planned (branch commit 2176dd8a).